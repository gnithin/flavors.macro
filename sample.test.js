const pluginTester = require('babel-plugin-tester')
const plugin = require("babel-plugin-macros");

pluginTester({
    plugin,
    snapshot: true,

    babelOptions: { filename: __filename },
    tests: [
        `
      import flavors from './flavors'
      flavors();
    `,
    ],
})
