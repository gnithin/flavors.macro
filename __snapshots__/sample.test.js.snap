// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`macros 1. macros: 1. macros 1`] = `
"
import flavors from './flavors'
flavors();

      ↓ ↓ ↓ ↓ ↓ ↓

import flavors from './flavors';
flavors();
"
`;
